hw/application_fpga/core/uds/README.md

```diff
@@ -31,4 +31,15 @@ Any access to another address will be ignored by the core.
 
 ## Implementation
 
-The UDS words are implemented using discrete registers.
+The 8 UDS words are stored using 32 named LUT4s identified by
+"uds\_rom\_idx".
+
+We use bit indexing from all of the 32 LUTs to store each bit of the
+word, i.e., the first UDS word consists of bit 0 from each 32 LUTs,
+the second word consists of bit 1 from each of the 32 LUTs, and so
+on... for a maximum storage of 16 words.
+
```

Comment:
See, synch with README for the UDI. This is slightly different from what is proposed for the UDI.